### PR TITLE
sync with upstream master

### PR DESCRIPTION
**Pull request by devilleweppenaar**
This also includes the `redigo` changes in #3.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,9 @@
-module github.com/agilebits/redicrypt
+module github.com/cohix/redicrypt
 
 go 1.14
 
 require (
-	github.com/gomodule/redigo v2.0.0+incompatible
```
Comment (grellyd) [inline go.mod:6]: Sorry are we downgrading? 🤔

**Review by grellyd — APPROVED** [review]
Looks fine to me! That one question re versions, but otherwise looks fine.

File: go.mod (hunk 1 above)
Comment (devilleweppenaar) [reply go.mod:6]: I think we're actually upgrading:
Screen Shot 2021-03-25 at 2 08 15 PM

Comment (devilleweppenaar) [reply go.mod:6]: I wasn't sure, so I checked and found this answer on [stackoverflow](https://stackoverflow.com/a/57356777):

> +incompatible means the dependency has a semver major version of 2 or higher and is not a Go module yet (it doesn't have go.mod in its source code).

Comment (grellyd) [reply go.mod:6]: Ah I understand. Thank you for clarifying!

**Pull request merged**